web/packages/hovercards/src/core.ts

```diff
@@ -295,32 +366,227 @@ export default class Hovercards {
 				<div class="gravatar-hovercard__body">
 					${ description ? `<p class="gravatar-hovercard__description">${ escHtml( description ) }</p>` : '' }
 				</div>
+				<div class="gravatar-hovercard__social-links">
+					<img class="gravatar-hovercard__verified-icon" src="https://secure.gravatar.com/icons/verified.svg" width="24" height="24" title="Verified accounts" alt="Verified icon">
+					<a class="gravatar-hovercard__social-link" href="${ trackedProfileUrl }" target="_blank" data-service-name="gravatar">
+						<img class="gravatar-hovercard__social-icon" src="https://secure.gravatar.com/icons/gravatar.svg" width="32" height="32" alt="Gravatar" />
+					</a>
+					${ renderSocialLinks }
+				</div>
+				${ ctaButtons }
 				<div class="gravatar-hovercard__footer">
-					<div class="gravatar-hovercard__social-links">
-						<a class="gravatar-hovercard__social-link" href="${ trackedProfileUrl }" target="_blank" data-service-name="gravatar">
-							<img class="gravatar-hovercard__social-icon" src="https://secure.gravatar.com/icons/gravatar.svg" width="32" height="32" alt="Gravatar" />
-						</a>
-						${ renderSocialLinks }
-					</div>
+					<span class="gravatar-hovercard__profile-url" title="${ profileUrl }">${ profileUrl.replace( 'https://', '' ) }</span>
 					<a
 						class="gravatar-hovercard__profile-link${ isEditProfile ? ' gravatar-hovercard__profile-link--edit' : '' }"
 						href="${ isEditProfile ? 'https://gravatar.com/profiles/edit?utm_source=hovercard' : trackedProfileUrl }"
 						target="_blank"
 					>
 						<span class="gravatar-hovercard__profile-link-text">
-							${ isEditProfile ? __( i18n, 'Edit your profile' ) : __( i18n, 'View profile' ) }
+							${ isEditProfile ? __( i18n, 'Edit your profile →' ) : __( i18n, 'View profile →' ) }
 						</span>
-						<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
-							<path d="M12.6667 8.33338L9.16666 12.1667M12.6667 8.33338L2.66666 8.33338M12.6667 8.33338L9.16666 4.83338" stroke-width="1.5"/>
-						</svg>
 					</a>
 				</div>
+				${ contactsDrawer }
+				${ sendMoneyDrawer }
+				${ backgroundColor ? '<div class="gravatar-hovercard__profile-color"></div>' : '' }
 			</div>
-    	`;
+		`;
+
+		const hovercardInner = hovercard.querySelector( '.gravatar-hovercard__inner' );
+		const headerImageEl = hovercardInner.querySelector< HTMLElement >( '.gravatar-hovercard__header-image' );
+		const profileColorEl = hovercardInner.querySelector< HTMLElement >( '.gravatar-hovercard__profile-color' );
```

Comment:
Nit: Consider specifying a more precise element type:

```suggestion
		const headerImageEl = hovercardInner.querySelector< HTMLDivElement >( '.gravatar-hovercard__header-image' );
		const profileColorEl = hovercardInner.querySelector< HTMLDivElement >( '.gravatar-hovercard__profile-color' );
```